wagtail/admin/static_src/wagtailadmin/scss/userbar.scss

```diff
@@ -151,6 +152,11 @@ $positions: (
         width: auto;
         margin: 0;
     }
+
+    &:focus {
+        border-color: $color-black;
+        outline: none;
+    }
```

Comment:
As it turns out this is slightly more complicated than I expected. It doesn't appear to be possible to manually define two-tone colors in an outline, so we are forced to rely in browser-specific properties.

In 5739d6bec0ccdfd3ce4ced45d57df49b21ca0784 I've changed the focus style to use `-webkit-focus-ring-color` which works fine for Webkit-based browsers. The two-tone outline appears to be specific to Chrome however.
```css
&:focus {
        outline: 5px auto -webkit-focus-ring-color;
    }
```

Chrome:
![image](https://user-images.githubusercontent.com/13856515/120921887-5c579500-c6c6-11eb-8280-0c541a3ccb09.png)

Safari:
![image](https://user-images.githubusercontent.com/13856515/120921664-151cd480-c6c5-11eb-88ee-57838d1dbe15.png)


Firefox is a not based on webkit and doesn't support `-webkit-focus-ring-color`. The closest equivalent I could find is `-moz-mac-focusring` (source: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value) which appears to be specific to Mac.

Firefox (on Mac, using `-moz-mac-focusring`):
![image](https://user-images.githubusercontent.com/13856515/120921966-c3754980-c6c6-11eb-964a-2cb9a0f8bd6d.png)


I'm going to do some more research on other viable options. If you (or anyone else) have any suggestions please feel free to share them!

Reply:
Looks like specifying `outline: auto` forces the browser to display an outline AND uses the default outline style provided by the browser.

I found that specifying an `outline-offset` helps make the outline more visible. 

Google Chrome on Mac (`outline: auto` and `outline-offset: 5px`)
![image](https://user-images.githubusercontent.com/13856515/120922568-359b5d80-c6ca-11eb-843a-7c16a7025846.png)

Reply:
Pushed cbf6b8f295d710ce8b0a9942e4c4dc61ecdb317e with the outline: auto and outline-offset change specified above.

How do you feel about this @thibaudcolas? Do you think this will work?

Here is Firefox on Mac with this change applied:

![image](https://user-images.githubusercontent.com/13856515/120922734-181ac380-c6cb-11eb-87ba-6d1bde6c9565.png)

Firefox on Windows uses a different focus style which I find to be not very visible.

![image](https://user-images.githubusercontent.com/13856515/120922800-7e9fe180-c6cb-11eb-97a1-a413779c88a6.png)

Reply:
As discussed during the accessibility team meeting on 2021-06-11, I've changed this to use the thick yellow (orange?) outline used elsewhere in Wagtail. Marking this as resolved.

![image](https://user-images.githubusercontent.com/13856515/122642016-b4e85280-d108-11eb-865b-f11422dd228f.png)